### PR TITLE
Fixing changes in ChatGPT web interface

### PR DESCRIPTION
**Pull request by micz**
SendButton CSS selector fixed.
Stripping only one _ChatGPT_ in the response (it was _ChatGPTChatGPT_).

### CODE REVIEW

**Comment by micz** [comment]
@adamlui are they changing the web interface always so often?

**Review by adamlui — CHANGES_REQUESTED** [review]
hey @micz I didn't get the UI update yet so `button[class*="cursor"][class*="bottom"]` is still the working selector for me, but can you inspect it and see if it has `data-testid="send-button"`? If so, you should use that instead because the class is selecting the close button to an alert my GPT-4o usage is depleted

**Comment by adamlui** [comment]
> @adamlui are they changing the web interface always so often?

Not usually but lately I've made frequent design changes to my https://github.com/adamlui/chatgpt-apps and their programmers and designers tend to follow my activity (notice they changed favicon to black/white, that came shortly after I updated the robot emoji to black/white OpenAI icon to represent ChatGPT in https://github.com/KudoAI/chatgpt.js#-contributors) so that is why all the sudden updates lately it is my fault

Since you have the new UI, the `chatgpt.isLoaded()` needs updating too because the sidebar changed so `nav button[id*="menu"]` no longer exists, could you inspect the menu button for new attr maybe? (in separate PR is best)

**Comment by micz** [comment]
> hey @micz I didn't get the UI update yet so `button[class*="cursor"][class*="bottom"]` is still the working selector for me, but can you inspect it and see if it has `data-testid="send-button"`? If so, you should use that instead because the class is selecting the close button to an alert my GPT-4o usage is depleted

This is what I see:

`<button disabled="" class="mb-1 mr-1 flex h-8 w-8 items-center justify-center rounded-full bg-black text-white transition-colors hover:opacity-70 focus-visible:outline-none focus-visible:outline-black disabled:bg-[#D7D7D7] disabled:text-[#f4f4f4] disabled:hover:opacity-100 dark:bg-white dark:text-black dark:focus-visible:outline-white disabled:dark:bg-token-text-quaternary dark:disabled:text-token-main-surface-secondary"><svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="none" viewBox="0 0 32 32" class="icon-2xl"><path fill="currentColor" fill-rule="evenodd" d="M15.192 8.906a1.143 1.143 0 0 1 1.616 0l5.143 5.143a1.143 1.143 0 0 1-1.616 1.616l-3.192-3.192v9.813a1.143 1.143 0 0 1-2.286 0v-9.813l-3.192 3.192a1.143 1.143 0 1 1-1.616-1.616z" clip-rule="evenodd"></path></svg></button>`

**Comment by micz** [comment]
> Since you have the new UI, the `chatgpt.isLoaded()` needs updating too because the sidebar changed so `nav button[id*="menu"]` no longer exists, could you inspect the menu button for new attr maybe? (in separate PR is best)

I'm not using it, I use a few functions [here](https://github.com/micz/ThunderAI/blob/6778619da27598fec35ef54cc3a4ff0b19b17aaa/js/mzta-chatgpt.js). It's a Thunderbird add-on, I wait for the new window to load and then check for the login, you can see it [here](https://github.com/micz/ThunderAI/blob/6778619da27598fec35ef54cc3a4ff0b19b17aaa/js/mzta-chatgpt.js#L262).

This is the new menu button: `<button class="flex h-10 w-10 items-center justify-center rounded-full hover:bg-token-main-surface-secondary focus-visible:bg-token-main-surface-secondary focus-visible:outline-0" type="button" id="radix-:ra:" aria-haspopup="menu" aria-expanded="true" data-state="open" aria-controls="radix-:rb:">`

**Comment by adamlui** [comment]
@micz ok unless you can find some common `path[d*=""]` in the old button's child svg (still displayed to many):

```js
<path fill="currentColor" fill-rule="evenodd" d="M12 3a1 1 0 0 1 .707.293l7 7a1 1 0 0 1-1.414 1.414L13 6.414V20a1 1 0 1 1-2 0V6.414l-5.293 5.293a1 1 0 0 1-1.414-1.414l7-7A1 1 0 0 1 12 3" clip-rule="evenodd"></path>
```

...then the best/shortest return would probably be

```js
    document.querySelector('[data-testid="send-button"]') // pre-GPT-4o
 || document.querySelector('path[d*="M15.192 8.906a1.143"]').parentNode.parentNode.click() // post-GPT-4o
```

**Comment by adamlui** [comment]
> This is the new menu button: `<button class="flex h-10 w-10 items-center justify-center rounded-full hover:bg-token-main-surface-secondary focus-visible:bg-token-main-surface-secondary focus-visible:outline-0" type="button" id="radix-:ra:" aria-haspopup="menu" aria-expanded="true" data-state="open" aria-controls="radix-:rb:">`

I registered a new dummy acct upon the recommendation of a buddy and got the new UI, so I can fix the `chatgpt.isLoaded()` so don't worry bout that one

File: chatgpt.js [hunk 1]
```diff
@@ -1455,7 +1455,7 @@ const chatgpt = { // eslint-disable-line no-redeclare
                     );
                     response = responseDivs[nthOfResponse - 1].textContent;
                 }
-                response = response.replace(/^ChatGPTChatGPT/, ''); // strip sender name
+                response = response.replace(/^ChatGPT/, ''); // strip sender name
```
Comment (adamlui) [inline chatgpt.js:1458]: Actually 'ChatGPTChatGPT' was required previously, did you test and find out it is no longer required?

Comment (adamlui) [reply chatgpt.js]: I tested the change works w/ new UI but not the old so I added fallback regex to accomodate both

**Review by adamlui — APPROVED** [review]

**Comment by micz** [comment]
@adamlui I see you fixed it. Thanks!

**Pull request merged**